### PR TITLE
fixed student can add course function

### PR DESCRIPTION
**Pull request by Nouraal21**
fixed the student_can_add_course function and renamed the url

### CODE REVIEW

File: courses/views.py [hunk 1]
```diff
@@ -144,13 +145,10 @@ def remove_student_from_course(request, course_id, student_id):
...
 def student_can_add_course(request, course_id, student_id):
-    # this function is implemented incorrectly
-    # it should check the student_registration_open
-    # on course model, noura, open a ticket and fix this
-    if 'student_registration_open' in request.GET:
+    if Course.objects.get(pk=course_id).student_registration_open:
         course = Course.objects.get(pk=course_id)
         student = Student.objects.get(pk=student_id)
-        student.courses.add(course)
+        course.students.remove(student)
```
Comment (almarzouq) [inline courses/views.py]: This is incorrect
Can u see me tomorrow?

Comment (almarzouq) [inline courses/views.py]: I meant Sunday... Can u see me Sunday?

Comment (Nouraal21) [inline courses/views.py]: Oh sorry it should be add not remove

Comment (Nouraal21) [inline courses/views.py]: Yes sure at 8 a.m ?

Comment (almarzouq) [inline courses/views.py]: U are correct it should be add

Also, there needs to be an if statement
The student can only register if student_registration_open is true... It is a field on the student object

Comment (almarzouq) [inline courses/views.py]: If u understand me and can fix it, then no need to see me

**Comment by Nouraal21** [comment]
I will try to fix it if couldn't I will come Sunday inshallah

File: courses/views.py [hunk 2]
```diff
@@ -144,13 +145,12 @@ def remove_student_from_course(request, course_id, student_id):
...
-    # this function is implemented incorrectly
-    # it should check the student_registration_open
-    # on course model, noura, open a ticket and fix this
-    if 'student_registration_open' in request.GET:
-        course = Course.objects.get(pk=course_id)
-        student = Student.objects.get(pk=student_id)
-        student.courses.add(course)
+    course = Course.objects.get(pk=course_id)
+    student = Student.objects.get(pk=student_id)
+    if course.student_registration_open is True:
```
Comment (almarzouq) [inline courses/views.py]: fix this line, change it to
    if course.student_registration_open:

Comment (Nouraal21) [inline courses/views.py]: fixed it now

**Pull request merged**